PennMobile/Home/Controllers/HomeViewController.swift

```diff
@@ -262,6 +262,7 @@ extension HomeViewController : DiningCellSettingsDelegate {
             diningItem.venues = DiningDataStore.shared.getVenues(for: DiningVenue.defaultVenueIds)
         } else {
             diningItem.venues = DiningDataStore.shared.getVenues(for: venueIds)
+            diningItem.venueIds = venueIds
```

Comment:
This makes sense but let's do the same for the other side of the if statement if the selected venue ids array is empty. In that case, we want to display the default venue ids.